<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerForm</class>
 <widget class="QDialog" name="WorkerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Añadir Nuevo Trabajador</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_nombre">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_nombre"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_apellido1">
       <property name="text">
        <string>Primer Apellido:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_apellido1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_apellido2">
       <property name="text">
        <string>Segundo Apellido:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_apellido2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_rol">
       <property name="text">
        <string>Rol:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="combo_rol"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_cargo">
       <property name="text">
        <string>Cargo Específico:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_cargo"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_superior">
       <property name="text">
        <string>Superior Directo:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="combo_superior"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>